resources/web/wwi/FloatingWindow.js

```diff
@@ -180,11 +180,13 @@ export default class FloatingWindow {
         left = fw.offsetLeft + dX;
         if (top < 0 || posY < topOffset) // top boundary
           top = 0;
-        else if (top + fw.offsetHeight > containerHeight || posY > containerHeight - fw.offsetHeight + topOffset) // bottom boundary
+        // bottom boundary
+        else if (top + fw.offsetHeight > containerHeight || posY > containerHeight - fw.offsetHeight + topOffset)
```

Comment:
Just a minor question, isn't better to move the comment inside the corresponing if block instead of the previous one?
```suggestion
        else if (top + fw.offsetHeight > containerHeight || posY > containerHeight - fw.offsetHeight + topOffset)
          // bottom boundary
```

Reply:
Yes